modules/UpgradeWizard/uw_utils.php

```diff
@@ -2248,42 +2256,54 @@ function unlinkUploadFiles()
 //	}
 }
 
+/**
+ * Recursively deletes a directory tree.
+ *
+ * @param string $folder         The directory path.
+ * @param bool   $keepRootFolder Whether to keep the top-level folder.
+ *
+ * @return bool TRUE on success, otherwise FALSE.
+ */
```

Comment:
Spacing:

```
/**
 * Recursively deletes a directory tree.
 *
 * @param string $folder The directory path.
 * @param bool $keepRootFolder Whether to keep the top-level folder.
 *
 * @return bool TRUE on success, otherwise FALSE.
 */
```